mobile/README.md

```diff
@@ -1,104 +1,261 @@
-# Xmartlabs' Flutter template
+<img src="icons/splash_logo.png" width=50%/>
 
-## Arch Overview 
+# Workshop: Exploring the Serverless World, An Introduction with Supabase.
 
-The project is divided into two main folders:
-- The UI contains all app screens.
-- The Core contains the models and the data layer.
+## Workshop Details:
 
-### UI section
+In this workshop, you will learn how to build a cross-platform mobile application using the Flutter framework and integrate it with Supabase.
 
-[Flutter Bloc][bloc] is used for state management, specifically, we use Cubit to manage the screen state.
-Each app section is added in a project folder which contains three components, the Screen (a `StatelessWidget`, the UI), the Cubit and the state.
+The workshop will explore:
+- Creation and Configuration of a Project in Supabase
+- Authentication
+- Implementing Messaging Functions
 
-The `MainScreen` is the Widget that contains all screens. It defines the `MaterialApp` and provides the app router.
-The router has two subgraphs, the `UnauthenticatedRouter` used for unauthenticated users and the `AuthenticatedRouter` used for authenticated users.
+## Workshop requirements
 
-The [app router][app_router] is provided by [auto_route][auto_route], and contains the previous sections with some nested screens.
+To participate on this workshop, the only two requirements you need are having basic knowledge of Flutter and Dart programming and a code editor (e.g., Visual Studio Code, Android Studio, or IntelliJ IDEA) installed in your machine. 
```

Comment:
```suggestion
```suggestion
To effectively engage with the Xmartchat demo app and our Supabase workshop, please ensure you have the following:

- *Flutter Installation:* Make sure you have Flutter installed on your system. If not, you can follow the official Flutter installation guide: [Flutter Installation Guide](https://flutter.dev/docs/get-started/install)

- *Dart & Flutter Knowledge:* Familiarity with Dart programming language and Flutter framework is essential. If you are new to Dart and Flutter, consider going through the official documentation and tutorials to strengthen your skills.

- *Integrated Development Environment (IDE):* You'll need an IDE for Flutter development. We recommend using either Visual Studio Code (VS Code) or Android Studio. Choose the one you are most comfortable with and ensure it's properly set up for Flutter development.
``````